SteamList/Controllers/FavoritesViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  FavsViewController.swift
+//  SteamList
+//
+//  Created by Macbook on 27/12/2022.
+//
+
+import UIKit
+
+final class FavoritesViewController: UIViewController {
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureTableView()
+        configureLeftBarButton()
+        configureRightBarButton()
+    }
+    
+    private func configureRightBarButton() {
+        navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Edit", style: .plain, target: self, action: nil)
+        navigationItem.rightBarButtonItem?.tintColor = .white
+    }
+    
+    private func configureLeftBarButton() {
+        navigationItem.leftBarButtonItem = UIBarButtonItem(title: "Sort", style: .plain, target: self, action: nil)
+        navigationItem.leftBarButtonItem?.tintColor = .white
+    }
+    
+    private func configureTableView() {
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(UINib.init(nibName: FavoritesTableViewCell.cellId, bundle: nil), forCellReuseIdentifier: FavoritesTableViewCell.cellId)
+    }
+}
+
+extension FavoritesViewController: UITableViewDelegate {
+}
+
+extension FavoritesViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return MockData.games.filter {game in game.isFavorite}.count
```

Comment:
I would suggest to hold a property inside your view controller instead of accessing the shared instance globally.

```
private let games = MockData.games
```
then, you can easily filter it in this method